src/ledger/readme.md

```diff
@@ -57,7 +57,7 @@ For more information on Shreds, see the spec [here](https://github.com/solana-fo
 
 The ShredCollector is responsible for gathering and storing shreds from the network. While it is not a direct 
```

Comment:
Nit: Should this section of the documentation be renamed to emphasis Shred Network instead of ShredCollector? Or maybe rephrased to emphasis the task of collecting shreds from the network by the shred network module?